blackbox/sqllogictest/src/sqllogictest.py

```diff
@@ -283,9 +273,9 @@ def get_logger(level, filename=None):
     return logger
 
 
-def run_file(filename, hosts, log_level, log_file, failfast):
+def run_file(filename, host, port, log_level, log_file, failfast):
     logger = get_logger(log_level, log_file)
-    conn = connect(hosts)
+    conn = psycopg2.connect("host=" + host + " port=" + port + " dbname='' user='doc' password='passwd'")
```

Comment:
I think it would also be possible to set dbname to 'doc' and remove user+password